### ISSUE TITLE
upsert not working without -c switch

### ISSUE THREAD

**Issue by morriswinkler-simplesurance** [ForceCLI/force#453]
can not run upsert without -c

```
█ force bulk upsert
ERROR: Unknown command - upsert.
```

versus 

```
██ force bulk -c=upsert
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/heroku/force/command.runBulk2(0x16f9040, 0xc4200100e0, 0x0, 0x0)
	/Users/morris/go/src/github.com/heroku/force/command/bulk.go:152 +0x40d
github.com/heroku/force/command.runBulk(0x16f9040, 0xc4200100e0, 0x0, 0x0)
	/Users/morris/go/src/github.com/heroku/force/command/bulk.go:210 +0x3ba
main.main()
	/Users/morris/go/src/github.com/heroku/force/main.go:30 +0x2e2
```

any idea why https://github.com/heroku/force/blob/b95cf1de07a6d0627446cfcac9dbcd9af8996007/command/bulk.go#L151

holds a case for upsert but

https://github.com/heroku/force/blob/b95cf1de07a6d0627446cfcac9dbcd9af8996007/command/bulk.go#L223

doesn't?

**Comment by slancio** [comment]
I think because I wasn't thorough enough when I was working on adding upsert to the ordered bulk commands.  https://github.com/ForceCLI/force/commits?author=slancio

**Comment by morriswinkler-simplesurance** [comment]
👍

**Comment by slancio** [comment]
https://github.com/ForceCLI/force/pull/476  Later than I said, but here it is.

**Comment by cwarden** [comment]
Fixed in #476.  Thanks, @slancio.
